x/apps/keeper/querier_test.go

```diff
@@ -30,7 +31,7 @@ func Test_queryApplications(t *testing.T) {
 		want  []byte
 		want1 sdk.Error
 	}{
-		{"Test query applicaitons", args{
+		{"Test query applications", args{
```

Comment:
I grepped the code and found three more places with this typo.  Can you correct those too?

```
$ grep -inr applicaitons --include=*.go
x/apps/types/errors_test.go:63:func TestError_ErrApplicaitonStatus(t *testing.T) {
x/apps/keeper/appStateChanges_test.go:56:func TestAppStateChange_ValidateApplicaitonStaking(t *testing.T) {
x/apps/keeper/querier_test.go:47:                               t.Errorf("queryApplicaitons() got1 = %v, want %v", got1, tt.want1)
```

Reply:
Ah, my bad.  I only grepped a plural "applicaitons".  There are more singular ones.

Actually I don't care these typos at all.  It's totally fine to leave them.  This PR is to explain AAT, not a boring typo-fixing work.  Instead of correcting them, it would be good to file a new issue and forget :)

```
$ grep -inr applicaiton
app/cmd/cli/query.go:538:               params := rpc.HeightAndApplicaitonOptsParams{
app/cmd/rpc/rpc_test.go:643:    var params = HeightAndApplicaitonOptsParams{
app/cmd/rpc/rpc_test.go:659:    params = HeightAndApplicaitonOptsParams{
app/cmd/rpc/query.go:78:type HeightAndApplicaitonOptsParams struct {
app/cmd/rpc/query.go:732:       var params = HeightAndApplicaitonOptsParams{}
x/apps/types/errors_test.go:163:func TestError_ErrApplicaitonPubKeyExists(t *testing.T) {
x/apps/types/errors_test.go:238:func TestError_ErrBadApplicaitonAddr(t *testing.T) {
x/apps/types/errors_test.go:258:                                t.Errorf("ErrBadApplicaitonAddr(): returns %v but want %v", got, tt.want)
x/apps/keeper/application_test.go:40:                           t.Errorf("Applicaiton.GetApplication() = got %v, want %v", found, tt.want)
x/apps/keeper/application_test.go:155:                          t.Errorf("Applicaiton.GetAllApplications() = got %v, want %v", got, tt.want)
x/apps/keeper/application_test.go:184:                          t.Errorf("Applicaiton.GetAllApplications() = got %v, want %v", got, tt.want)
x/apps/keeper/appStateChanges_test.go:15:func TestAppStateChange_ValidateApplicaitonBeginUnstaking(t *testing.T) {
x/apps/keeper/appUnstaked.go:19:// deleteUnstakingApplicaiton - DeleteEvidence an application address from the unstaking queue
```

Reply:
https://github.com/pokt-network/pocket-core/issues/1601